service/static/index.html

```diff
@@ -20,11 +21,24 @@
   </header>
   <main class="container py-3">
     <h1>Manage wishlists</h1>
-    <form>
+    <div>
```

Comment:
This is because if we use ```<form>...</form>```, the page will be refreshed if the user click the button in that form. 
I am kind of confused about that. I searched this in google and everything works well after I changed this part.
So if there are any better solutions, please let me know.

Reply:
Try using event.preventDefault() in the function that has logic of submit (event is a variable coming from arguments of that function)